src/Client/Util.java

```diff
@@ -46,4 +55,63 @@ public static String byteHexString(byte[] data)
 			ret += String.format("%02x", data[i]);
 		return ret;
 	}
+	
+    public static int[] getPop() {
+    	if (worldPopArray == null) worldPopArray = new int[6];
+    	if (!(System.currentTimeMillis() >= lastPopCheck + 60000)) {
+    		return worldPopArray;
+    	}
+    	lastPopCheck = System.currentTimeMillis();
+    	
+    	URL url;
+    	URLConnection con;
+    	InputStream is = null;
+    	BufferedReader br;
+    
+    	try {
+	        url = new URL("http://www.runescape.com/classicapplet/playclassic.ws");
+	        con = url.openConnection();
+	        is = con.getInputStream();
+	        br = new BufferedReader(new InputStreamReader(is));
+    	} catch (IOException ioe) {
+    		System.err.println("Network connection issues");
```

Comment:
Could this System.err use the Logger class instead? Also maybe Logger.Error and Fatal should use System.err.println instead of System.out.println.